components/statewide-header/index.css

```diff
@@ -161,20 +161,10 @@
   display: none;
 }
 
-.mobile-only {
```

Comment:
Got it, I ran npm run build in the statewide-header directory and made another PR. Let me know if this works?

Also to clarify anytime I make a change in the scss file, I should do npm run build in the components directory so that the changes are made in the plain css? Is that the purpose of doing npm run build?

Reply:
Right